extensions/amp-image-slider/0.1/amp-image-slider.js

```diff
@@ -153,76 +159,77 @@ export class AmpImageSlider extends AMP.BaseElement {
     this.registerAction('seekTo', invocation => {
       const {args} = invocation;
       if (args) {
-        let value;
-        if (args['value'] !== undefined) {
-          value = args['value'];
-          user().assertNumber(value,
+        if (args['percent'] !== undefined) {
+          const percent = args['percent'];
+          user().assertNumber(percent,
               'value to seek to must be a number');
-        } else if (args['percent'] !== undefined) {
-          value = args['percent'];
-          user().assertNumber(value,
-              'percent to seek to must be a number');
-          value *= 0.01; // percentage to 0-1 value
-        }
-        if (value !== undefined) {
-          this.updatePositions_(value);
+          this.updatePositions_(percent);
         }
       }
     }, ActionTrust.LOW);
 
+    const initialPercentString = this.element.getAttribute('initial-percent');
     // TODO(kqian): move this before building child components on issue
     // This is the only step when content tree is attached to document
     return this.mutateElement(() => {
       this.element.appendChild(this.container_);
       // Ensure ampdoc exists on the amp-imgs
       this.leftMask_.appendChild(this.leftAmpImage_);
       this.rightMask_.appendChild(this.rightAmpImage_);
+      // Set initial positioning
+      if (initialPercentString) {
+        const initialPercent = Number(initialPercentString);
+        if (isNaN(initialPercent)) {
```

Comment:
We don't need this to be runtime validated because regex is something that validation actually can do.